<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="Title">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>File Explorer</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="fileName">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>File Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="filePath"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseButton">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QTextEdit" name="FileContent"/>
           </item>
           <item row="2" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>438</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="writeButton">
             <property name="text">
              <string>Write To  Document</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextBrowser" name="debugTextBrowser"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browseSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>writeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>writeDocSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filePath</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>returnPressedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseSlot()</slot>
  <slot>writeDocSlot()</slot>
  <slot>returnPressedSlot()</slot>
 </slots>
</ui>
